### PR TITLE
feat: 2023-10-17 14:12 - Update API specification

### PR DESCRIPTION
**Pull request by devopness-automations**
This PR is auto generated to update `spec.json` file.

### CODE REVIEW

File: packages/sdks/javascript/src/api/generated/apis/users-environments-api.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { ApiBaseService } from "../../../services/ApiBaseService";
+import { ApiResponse } from "../../../common/ApiResponse";
+import { ArgumentNullException } from "../../../common/Exceptions";
+import { ApiError } from '../../generated/models';
+import { EnvironmentRelation } from '../../generated/models';
+
+/**
+ * UsersEnvironmentsApiService - Auto-generated
+ */
+export class UsersEnvironmentsApiService extends ApiBaseService {
```
Comment (Silva97) [inline packages/sdks/javascript/src/api/generated/apis/users-environments-api.ts:23]: @brlipi Note that this `*Service` class file is new and won't be used in any service yet.

Take a look at the file `packages/sdks/javascript/src/services/UserService.ts` to see how these service classes are used. Look at the `passwords` property as a reference.

**Review by Silva97 — CHANGES_REQUESTED** [review]

File: packages/sdks/javascript/src/api/generated/apis/users-environments-api.ts (hunk 1 above)
Comment (brlipi) [reply packages/sdks/javascript/src/api/generated/apis/users-environments-api.ts:23]: `UsersEnvironmentsApiService` adicionado ao `UserService` no commit https://github.com/devopness/devopness/pull/770/commits/ea54d492e285c1fc1dafe1cd45a968eeff6ccae0

File: packages/sdks/javascript/src/services/UserService.ts [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import { UsersApiService } from '../api/generated/apis/users-api';
 import { UsersTeamInvitationsApiService } from '../api/generated/apis/users-team-invitations-api';
 import { UsersPasswordsApiService } from '../api/generated/apis/users-passwords-api';
+import { UsersEnvironmentsApiService } from '../api/generated/apis/users-environments-api';
 
 export class UserService extends UsersApiService {
     public teamInvitations = new UsersTeamInvitationsApiService();
     public passwords = new UsersPasswordsApiService();
+    public environments = new UsersEnvironmentsApiService();
 }
```
Comment (jfoliveira) [inline packages/sdks/javascript/src/services/UserService.ts:10]: I'd suggest keeping properties sorted alphabetically:
```suggestion
    public environments = new UsersEnvironmentsApiService();
    public passwords = new UsersPasswordsApiService();
    public teamInvitations = new UsersTeamInvitationsApiService();
}
```

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: packages/sdks/javascript/src/services/UserService.ts (hunk 2 above)
Comment (brlipi) [reply packages/sdks/javascript/src/services/UserService.ts]: I've rearranged it in alphabetical order: https://github.com/devopness/devopness/pull/770/commits/d09ccd007273be7ca2672d0e537464730b0d7d7c

**Review by jfoliveira — APPROVED** [review]

**Pull request merged**